components/Layout/FormStyles/StyledLabel/index.js

```diff
@@ -0,0 +1,12 @@
+import color from "../../Colors";
```

Comment:
you would not need to import the variables here if you would use css variables...